PROTOCOL.md

```diff
@@ -2,179 +2,287 @@
 
 Extreme portability is a key design goal of libchan.
 
-This document specifies the libchan protocol to allow multiple implementations to co-exist with
-full interoperability.
+This document specifies the libchan protocol to allow multiple implementations
+to co-exist with full interoperability.
 
 ## Version
 
-No version yet.
+0.2.0
 
-## Author
+## Authors
 
 Solomon Hykes <solomon@docker.com>
+Derek McGowan <derek@docker.com>
+Matteo Collina <matteo.collina@gmail.com>
 
 ## Status
 
-This specification is still work in progress. Things will change, probably in reverse-incompatible ways.
-We hope to reach full API stability soon.
+This specification is nearing a stable release. The protocol still may change in
+reverse-incompatible ways.
 
 ## Terminology
 
 ### Channel
 
-A `channel` is an object which allows 2 concurrent programs to communicate with each other. The semantics
-of a libchan channel are very similar (but not identical) to those of Go's native channels.
+A `channel` is an object which allows 2 concurrent programs to communicate with
+each other. The semantics of a libchan channel are very similar (but not
+identical) to those of Go's native channels. A channel may be used
+synchronously, but do not support Go's channel select semantics.
```

Comment:
I would add:

"For those not familiar with Go, this means it does not support any synchronization primitives".